src/client/Microsoft.Identity.Client/Platforms/uap/UapPlatformProxy.cs

```diff
@@ -107,7 +107,7 @@ protected override string InternalGetOperatingSystem()
         {
             // In WinRT, there is no way to reliably get OS version. All can be done reliably is to check
             // for existence of specific features which does not help in this case, so we do not emit OS in WinRT.
-            return null;
+            return "Windows 10";
```

Comment:
What is the reason for this change? Maybe should update the comment to match.

Reply:
Removed the comment. We should always send the OS as ESTS makes some decisions based on it,.